### PR TITLE
Modify the error message of server validation

### PR DESCRIPTION
**Pull request by Pengpengwanga**
Signed-off-by: Pengpeng Wang <pengpengw@vmware.com>

### CODE REVIEW

File: flowgate-api/src/main/java/com/vmware/flowgate/verifycert/VRopsAuth.java [hunk 1]
```diff
@@ -30,7 +30,7 @@ public VRopsAuth(SDDCSoftwareConfig config) {
                .useInternalApis(false).build().newClient();
       } catch (Exception e) {
          e.printStackTrace();
```
Comment (yixingjia) [inline flowgate-api/src/main/java/com/vmware/flowgate/verifycert/VRopsAuth.java:32]: log

File: flowgate-api/src/main/java/com/vmware/flowgate/service/ServerValidationService.java [hunk 2]
```diff
@@ -59,26 +59,21 @@ public void validateVROServer(SDDCSoftwareConfig server) {
...
          ss.getClient().apiVersionsClient().getCurrentVersion();
-      } catch (AuthException e) {
+      } catch (AuthException authException) {
          throw new WormholeRequestException(HttpStatus.BAD_REQUEST,
-               "Invalid user name or password.Please check your userName or password.", e.getCause());
-      } catch (Exception e3) {
-         if(e3.getCause() instanceof UndeclaredThrowableException) {
-            UndeclaredThrowableException e = (UndeclaredThrowableException)e3.getCause();
-            if (e.getUndeclaredThrowable().getCause() instanceof ConnectException) {
+               "Invalid user name or password.Please check your userName or password.", authException.getCause());
```
Comment (yixingjia) [inline flowgate-api/src/main/java/com/vmware/flowgate/service/ServerValidationService.java:64]: remove "Please check your userName or password."

File: flowgate-api/src/main/java/com/vmware/flowgate/verifycert/VRopsAuth.java (hunk 1 above)
Comment (Pengpengwanga) [reply flowgate-api/src/main/java/com/vmware/flowgate/verifycert/VRopsAuth.java]: removed the catch block

File: flowgate-api/src/main/java/com/vmware/flowgate/service/ServerValidationService.java (hunk 2 above)
Comment (Pengpengwanga) [reply flowgate-api/src/main/java/com/vmware/flowgate/service/ServerValidationService.java]: Removed

**Pull request merged**